app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -547,7 +547,9 @@ def os_name_mapping(host)
     end
 
     def arch_name_mapping(host)
-      (host.architecture.name == 'x86_64') ? 'x64' : host.architecture.name.downcase if host.architecture
```

Comment:
This look a bit strange and is maybe a case for the safe navigation operator, but I don't know the original background...